src/bootstrap-table.js

```diff
@@ -1911,7 +1911,8 @@
             that.trigger(e.type === 'click' ? 'click-row' : 'dbl-click-row', item, $tr, field);
 
             // if click to select - then trigger the checkbox/radio click
-            if (e.type === 'click' && that.options.clickToSelect && column.clickToSelect) {
+            var ignoreTags = [ "A", "BUTTON" ];
```

Comment:
I think this can be consider as table option.

Reply:
Ok! I implemented this with an optional callback `ignoreClickToSelectOn` which takes an element and returns true or false. If you like this idea, how should I update the docs? I see `docs/_i18n/en/documentation/table-options.md`, but there are also Spanish and Chinese translations I don't feel comfortable trying to update =)